tests/e2e/scale/conftest.py

```diff
@@ -0,0 +1,28 @@
+import logging
+from ocs_ci.framework import config
+
+log = logging.getLogger(__name__)
+
+
+def pytest_collection_modifyitems(items):
+    """
+    A pytest hook to skip certain tests when running on vms.
```

Comment:
what the vms is stand for here?   Virtual Machines?   If so it should be VMs or whole Virtual Machines.

Reply:
changed to "virtual machine"